### ISSUE TITLE
txt request

### ISSUE THREAD

**Issue by liustu** [xuchen-ethz/continuous_view_synthesis#2]
I have studied your article named recently. Could you please provide TXT files of training data and test data.Thanks for your reply.

**Comment by xuchen-ethz** [comment]
We follow the train/test split as Multi-view to Novel view. The model ids can be found at https://github.com/shaohua0116/Multiview2Novelview/tree/master/datasets/shapenet

**Comment by liustu** [comment]
I have read the setting of the code of this paper, but the location of this paper is the division of the training set and the verification set. Did you directly make the test set with the verification set, but did you not divide the verification set.&nbsp;




---Original---
